pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -220,7 +220,7 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     # Write pkgconfig aliases. Upstream report:
-    # https://github.com/xianyi/OpenBLAS/issues/1740
+    # https://github.com/OpenMathLib/OpenBLAS/issues/1740
```

Comment:
If I did not change this, there would be no rebuilds. Is it ok to leave in the old URL for less wasted CPU time?

Reply:
There's practically no waste on staging, as everything gets to rebuild anyway.